app/javascript/components/city-name.jsx

```diff
@@ -1,32 +1,13 @@
 import React from 'react';
-import { Query } from 'react-apollo';
-import gql from 'graphql-tag';
+import PropTypes from 'prop-types';
 import { WhiteTitleLarge } from '@components/typography';
-import { LoadingMessage, ErrorMessage } from '@messages/default-messages';
 
-const getCityName = gql`
-  {
-    primaryLocation {
-      cityName
-    }
-  }
-`;
-
-export const City = () => (
-  <Query query={getCityName}>
-    {({ loading, error, data }) => {
-      if (loading) {
-        return <LoadingMessage />;
-      }
-
-      if (error) {
-        return <ErrorMessage />;
-      }
-
-      const { cityName } = data.primaryLocation;
-      return cityName && <WhiteTitleLarge>{cityName}</WhiteTitleLarge>;
-    }}
-  </Query>
+export const City = ({ cityName }) => (
+  <WhiteTitleLarge>{cityName}</WhiteTitleLarge>
```

Comment:
Be careful here. Just because cityName is a string does not mean its display ready. It's probably fine here though. Do we need to care about rendering every string that come in the URL?

Also, this will render right away, instead of having to manage a loading state, which is a better experience for everyone involved. 👍